File: api/Entities/Deployments/DepItem.cs
namespace api.Entities.Deployments
{
    public class DepItem: BaseEntity
    {
        public DepItem()
        {
        }


        public int DepId { get; set; }
        public DateOnly TransactionDate { get; set; }   
        public int Sequence { get; set; }
        public int NextSequence { get; set; }
        public DateOnly NextSequenceDate { get; set; }
    }
}

File: client/src/app/_models/process/depItem.ts
export interface IDepItem
{
    id: number;
    depId: number;
    transactionDate: Date;
    sequence: number;
    nextSequence: number;
    nextSequenceDate: Date;
    fullPath: string;
}

export class DepItem implements IDepItem
{
    id=0;
    depId=0;
    transactionDate = new Date();
    sequence = 0;
    nextSequence = 0;
    nextSequenceDate = new Date();
    fullPath = "";
}